diffsims/utils/shape_factor_models.py

```diff
@@ -77,7 +77,122 @@ def sinc(excitation_error, max_excitation_error, minima_number=5):
     -------
     intensity : array-like or float
     """
+    fac = np.pi * minima_number / max_excitation_error
+    num = np.sin(fac * excitation_error)
+    denom = fac * excitation_error
+    return np.nan_to_num(
+            np.abs(np.divide(num, denom, out=np.zeros_like(num), where=denom != 0)),
+            nan=1,
+            )
 
-    num = np.sin(np.pi * minima_number * excitation_error / max_excitation_error)
-    denom = excitation_error
-    return np.abs(np.divide(num, denom, out=np.zeros_like(num), where=denom != 0))
```

Comment:
I'm not sure these are quiet the same?

Reply:
no they are not. The denominator is scaled by the factor `fac` so that the maximum at s=0 comes out to 1. In my opinion that's a bit nicer to plot and compare for example to the linear case. In any case it shouldn't matter since the intensities get rescaled to the maximum at a later stage, and each reflection is multiplied by this same factor, right?

Reply:
Makes perfect sense, it looked like a refactor (from the code) but I had forgot you said you were scaling everything to have unit intensity, in which case this is fine and good.